README.md

```diff
@@ -1,7 +1,36 @@
 # Coding standards
 
-UCL Advanced Research Computing Centre's recommended coding standards for research software projects.
-These are our default assumed style choices unless there are existing project- or domain-specific local style conventions.
+These are the default coding standards for research software projects used by the Centre for Advanced Research Computing at UCL.
+
+They represent our default style and standard choices that our projects should
+use if there are no other constraints. If you are working with us, we'll
+probably start using these standards and may point you to this page (👋).
+
+In the best case, this means that there is only one standard to learn per language.
+We hope this results in:
+  * 😊 less time thinking about style,
+  * 💪 faster onboarding of staff to new projects, and
+  * 🚀 faster review of code.
```

Comment:
maybe consistent configuration of tooling to ensure standards are being met at as well?

Reply:
Is just
```
* 🛠️ a more consistent tooling setup,
```
clear enough?

Reply:
Yeah like it